tests/scripts/all.sh

```diff
@@ -1114,6 +1114,12 @@ component_build_armcc () {
     armc6_build_test "--target=aarch64-arm-none-eabi -march=armv8.2-a"
 }
 
+component_build_ssl_hw_record_accel() {
+    msg "build: default config with MBEDTLS_SSL_HW_RECORD_ACCEL enabled"
+    scripts/config.pl set MBEDTLS_SSL_HW_RECORD_ACCEL
```

Comment:
Just to be sure, no need to set MBEDTLS_SSL_PROTO_DTLS as well?

Reply:
Found that MBEDTLS_SSL_PROTO_DTLS is set in the default configuration thus resolving this conversation.